dev/app/app.js

```diff
@@ -4,16 +4,12 @@ var angular = require('angular');
 require('angular-route');
 require('angular-sanitize');
 require('angularfire');
-/**
- * @ngdoc overview
- * @name freeCodeLondonersCampApp
- * @description
- * # freeCodeLondonersCampApp
- *
- * Main module of the application.
- */
 
-var dataSource = (require('./app.config')) ? require('./shared/services/local') : require('./shared/services/firebase');
+require('./assets/sass/main.scss');
```

Comment:
It is a bit strange but the `require` statement is how webpack knows to look for an asset. I was just using React as an example of how it can be used.